pkg/doc/did/doc.go

```diff
@@ -1004,3 +1011,30 @@ func BuildDoc(opts ...DocOption) *Doc {
 
 	return doc
 }
+
+func createDocumentLoader() ld.DocumentLoader {
+	loader := ld.NewCachingDocumentLoader(ld.NewRFC7324CachingDocumentLoader(&http.Client{}))
+
+	didV1Context, _ := ioutil.ReadFile(filepath.Clean(filepath.Join(
+		jsonldContextPrefix, "didv1.jsonld"))) //nolint:errcheck
```

Comment:
These files are not bundled in the final binary. This won't work unless the files are distributed together with the binary. I believe all that @troyronda suggested was to put them in separate `.go` files.

Reply:
Ah, I see! My bad, one moment, I will fix it.

Reply:
Added `contexts.go`, thank you for the catch @llorllale !